src/app/packages/core/models/case-map-position.model.ts

```diff
@@ -10,7 +10,15 @@ export interface CaseMapProjectedState {
 	zoom?: number;
 }
 
-export interface CaseMapPosition {
+export interface ICaseResolutionData {
+	center: [number, number];
+	refPoint1: [number, number];
+	refPoint2: [number, number];
+	mapResolution: number;
+}
+
+export interface ICaseMapPosition {
```

Comment:
replace to CaseMapPosition .big refactor for all case model interfaces will come later